ncm-network/src/main/perl/network.pm

```diff
@@ -60,6 +60,52 @@ my %ethtool_option_map = (
 
 my $ethtoolcmd = "/usr/sbin/ethtool";
 
+# These values are hardcoded in the 'ip' tool
+my %rt_builtins = (
+    "dsfield" => {},
+    "protos" => {
+        0 => "unspec",
+        1 => "redirect",
+        2 => "kernel",
+        3 => "boot",
+        4 => "static",
+        8 => "gated",
+        9 => "ra",
+        10 => "mrt",
+        11 => "zebra",
+        12 => "bird",
+        13 => "dnrouted",
+        14 => "xorp",
+        15 => "ntk",
+        16 => "dhcp",
+    },
+    "realms" => {},
+    "scopes" => {
+        0 => "global",
+        255 => "nowhere",
+        254 => "host",
+        253 => "link",
+        200 => "site",
+    },
+    "tables" => {
+        0 => "unspec",
+        255 => "local",
+        254 => "main",
+        253 => "default",
+    },
+);
+
+my %mask_to_prefix = (
```

Comment:
use `NetAddr::Ip` like https://github.com/quattor/configuration-modules-core/pull/1087/files#diff-90bc6b2e2a105ae5c62daf55ef95b2a7R1014

Reply:
Yes, that should likely happen here. We didn't want to introduce new dependencies when this code was originally written.